### PR TITLE
Align formatting

### PR DESCRIPTION
**Pull request by yinonavraham**
Align the formatting of a list (currently squashed into a single wrapped line).
Set code formatting, mainly for file names, similar to other locations in this file.

### CODE REVIEW

**Review by dthaler — APPROVED** [review]
Looks good, thanks!

I see the CLA needs to be signed before this can be merged though.

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ and running.
 ### api_test.exe
 This test exercises various eBPF user mode eBPF APIs, including those to load programs,
 enumerate maps and programs etc. This test requires the eBPF user mode service (EbpfSvc), and the
```
Comment (saxena-anurag) [inline docs/GettingStarted.md:162]: I think EbpfSvc should also be changed to `EbpfSvc.exe`

Comment (yinonavraham) [reply docs/GettingStarted.md:162]: Here I think it refers to the service name, not the binary file name `ebpfsvc.exe`.
But I can make this change as well if needed.

Comment (dthaler) [reply docs/GettingStarted.md:162]: I'm ok with it either way.  (Such a change does not seem to be part of "align formatting" as a subject though so I'm fine with the PR as is.)

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**